### PR TITLE
Improvement

### PR DESCRIPTION
**Pull request by zelibobla**
- Replaced custom and too short eslint rules file with big set of rules provided by JS community: https://github.com/standard/eslint-config-standard
- Removed TypeScript file due to it's not used anywhere
- Applied eslint rules to the files
No logic changed in this PR. Only strong linting applied.

### CODE REVIEW

**Comment by pilievOoyala** [comment]
@zelibobla LGTM. We came up with an eslint config but haven't gotten around to applying it to all the code repos.

https://github.com/ooyala/html5-skin/blob/master/.eslintrc
https://github.com/ooyala/html5-skin/blob/master/.eslintignore

Can you take a look and see how much work it will be to applies these settings? Hopefully you can autofix most of it.

File: lib/src/app/cast.js [hunk 1]
```diff
@@ -17,59 +19,73 @@ import '../styles/player.css';
...
+      logger.info('MediaManager:onLoad', event);
+      const isAutoplayDesired = event && event.data && event.data.autoplay;
+      if (isAutoplayDesired) {
+        Timer.setIdle();
+      }
+      castPlayer.setAsset(event.data.media.customData);
+      if (castPlayer.OOPlayer != null) {
+        const state = castPlayer.OOPlayer.getState();
+        castPlayer.notifySenders(state);
+        if (isAutoplayDesired) {
```
Comment (aeng7) [inline lib/src/app/cast.js]: The description for this PR said that there were no logic changes. This if statement looks to be a logic change, so just double checking if this was intentional.

File: lib/src/app/cast.js [hunk 2]
```diff
@@ -17,59 +19,73 @@ import '../styles/player.css';
...
+   * onPause overrides the default behaviour to setup the default timeout
+   * when an asset is being paused
+   * @param {cast.receiver.MediaManager.Event} event
+   */
+  mediaManager.onPause = (() => {
+    mediaManager.origOnPause = mediaManager.onPause;
+    return (event) => {
+      logger.info('MediaManager:onPause', event);
+      Timer.setIdle(Timer.TIMEOUT.PAUSED);
+      if (castPlayer.OOPlayer != null) {
```
Comment (aeng7) [inline lib/src/app/cast.js]: This also looks to be a logic change.

File: lib/src/app/idleTimer.js [hunk 3]
```diff
@@ -8,29 +8,29 @@ import * as logger from 'loglevel';
 import castManager from './receiverManager';
 
 const timeout = {
-    PAUSED: 1000 * 60 * 5, // 5 minutes
-    IDLE: 1000 * 60 * 10 // 5 minutes
+  PAUSED: 1000 * 60 * 5, // 5 minutes
+  IDLE: 1000 * 60 * 10, // 5 minutes
```
Comment (aeng7) [inline lib/src/app/idleTimer.js]: Dangling comma. There looks to be a new lint condition regarding this here:

https://github.com/ooyala/chromecast-sample-receiver/pull/61/files#diff-b0d0bf091f28af6d677bffef8898431bR4

Shouldn't it trigger on this?

File: lib/src/app/player.d.ts [hunk 4]
```diff
@@ -1,229 +0,0 @@
-declare namespace player {
```
Comment (aeng7) [inline lib/src/app/player.d.ts]: Is this no longer needed?

File: lib/template.html [hunk 5]
```diff
@@ -38,7 +38,12 @@
             playerBrandingId: "pleaseFillTheRightPlayerId",
             skin: {
                 inline: {
+                    startScreen: {
```
Comment (aeng7) [inline lib/template.html]: These look like new additions as well. Was this intentional or intended for another PR?

File: lib/src/app/player.d.ts (hunk 4 above)
Comment (zelibobla) [reply lib/src/app/player.d.ts]: I didn't find any place where TypeScript used. Good idea to apply it, but it's a big effort. It's still not done yet, so I suggest to leave this file in git history and get back to it if needed.

File: lib/src/app/cast.js (hunk 1 above)
Comment (zelibobla) [reply lib/src/app/cast.js]: Sorry, I've branched not from master occasionally. Rolled this back.

File: lib/src/app/cast.js (hunk 2 above)
Comment (zelibobla) [reply lib/src/app/cast.js]: This is just one more way to extend an object created in ugly way be a google team: https://developers.google.com/cast/docs/custom_receiver (method of the object had to be implemented via prototype; it this case extend would go soft; but google team done awkward object and suggests now a tricky way to extend it, oops).
Logic doesn't change here

File: lib/template.html (hunk 5 above)
Comment (zelibobla) [reply lib/template.html]: This is because I've branched from wrong branch as well. Sorry. Rolled back.

**Comment by zelibobla** [comment]
> @zelibobla LGTM. We came up with an eslint config but haven't gotten around to applying it to all the code repos.
> 
> https://github.com/ooyala/html5-skin/blob/master/.eslintrc
> https://github.com/ooyala/html5-skin/blob/master/.eslintignore
> 
> Can you take a look and see how much work it will be to applies these settings? Hopefully you can autofix most of it.

@pilievOoyala I've just synced suggested by me rules with the one you refer to. Little update is just 'error' instead of 'warn'. Give a look at updated .eslint in this repo please. The rules that you don't see there are located in community's defaults: https://github.com/standard/eslint-config-standard/blob/master/eslintrc.json

Current repository currently satisfies the updated .eslint with no errors.

File: lib/src/app/cast.js (hunk 2 above)
Comment (aeng7) [reply lib/src/app/cast.js]: Sounds good, thanks for the explanation!

**Comment by aeng7** [comment]
LGTM

**Pull request merged**